cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f08ac68",
+   "metadata": {},
+   "source": [
+    "# Itteratively Find Improved Consensus Labels for Multiannotator Data using Cleanlab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44a0555b",
+   "metadata": {},
+   "source": [
+    "This example shows how to improve consensus labels by combining the CROWDLAB algorithm with itteratively retraining a model. For an introductory tutorial on finding consensus labels with the multiannotator library see [Find Best Consensus Labels for Multiannotator Data using Cleanlab](linke to hui wen's). \n",
+    "\n",
+    "The following code uses the [cifar10h](linke cifar10h) multiannotator labeling dataset which is a collection of 2751 annotators each labeling 200 examples for all 10,000 test images of the original [cifar10](link cifar10) dataset but **any multiannotator classification image dataset should work with the code below**. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95724ad1",
+   "metadata": {},
+   "source": [
+    "## 1. Install and import required dependencies, build example folder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68b291ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from matplotlib import pyplot as plt\n",
+    "from datetime import datetime\n",
+    "\n",
+    "from utils.model_training import train_model\n",
+    "from utils.model_training import sum_xval_folds\n",
+    "from cleanlab.multiannotator import get_majority_vote_label\n",
+    "from cleanlab.multiannotator import get_label_quality_multiannotator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f3551b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "now = datetime.now() # Current date and time\n",
+    "experiment_path = \"./experiment_\" + str(int(now.timestamp()))\n",
+    "\n",
+    "if not os.path.exists(experiment_path):\n",
+    "    os.makedirs(experiment_path)\n",
+    "    print(\"Directory \" , experiment_path ,  \" Created \")\n",
+    "else:    \n",
+    "    print(\"Directory \" , experiment_path ,  \" already exists\")\n",
+    "\n",
+    "print(f'Experiment saved in {experiment_path}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "23c50596",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import cifar10 data for model training or download it yourself by calling cifar2png cifar10 ./data/cifar10 --name-with-batch-index\n",
```

Comment:
"Import" -> "Download"

Reply:
Why do you also mention:
`or download it yourself by calling cifar2png cifar10 ./data/cifar10 --name-with-batch-index`

Isn't the `wget` command already downloading the data?

Maybe you mean to say: 
Here's an alternative command to download the data:  `cifar2png cifar10 ./data/cifar10 --name-with-batch-index`

Reply:
Yeah `wget` takes care of that, honestly is this an important alternative option? I can drop it entirely